src/main/ConfigurationFileReader.cpp

```diff
@@ -7,7 +7,7 @@
 #include "Resident.h"
 #include "Foreigner.h"
 
-#define SEPARATOR ','
+const static char SEPARATOR = ',';
```

Comment:
El `const static` no es necesario en C++, `const` es `static` al nivel del namespace.

https://stackoverflow.com/questions/177437/what-does-const-static-mean-in-c-and-c

Reply:
Lo que quiero evitar es que la usen desde afuera, pero tampoco la quiero hacer un miembro privado de la clase. Sacarle el static no se si garantiza que no se vea fuera del modulo